src/main/java/ch/ivyteam/ivy/maven/bpm/test/IvyTestRuntime.java

```diff
@@ -0,0 +1,35 @@
+package ch.ivyteam.ivy.maven.bpm.test;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Properties;
+
+public class IvyTestRuntime
+{
+  public static final String RUNTIME_PROPS_RESOURCE="ivyTestRuntime.properties";
+  
+  public static interface Key
```

Comment:
Implemenation detail. Can we hide this somehow.

Reply:
Had to make it public so that the mojo integration test can access it. 
- moving the test into the same package does not improve the situation (integration tests can't live in all packages that are relevant)
- but I think it's not that important anyway. This fields are public yes, but users never use this code in plain java. But via maven goal configurations. So the raised visibility isn't really something the users can see somewhere.